apis/interfaces.go

```diff
@@ -17,28 +17,30 @@ limitations under the License.
 package apis
 
 import (
+	"context"
+
 	authenticationv1 "k8s.io/api/authentication/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
 // Defaultable defines an interface for setting the defaults for the
 // uninitialized fields of this instance.
 type Defaultable interface {
-	SetDefaults()
+	SetDefaults(context.Context)
```

Comment:
What is this context argument expected to be used for? Do we have an example?

Reply:
For example we have a request to make default revision timeout configurable by the operator.
ConfigMaps, attached to the context can be used to get the operator desired default.